web/app/view/ReportController.js

```diff
@@ -52,9 +55,15 @@ Ext.define('Traccar.view.ReportController', {
         Traccar.app.showReports(false);
     },
 
+    init: function () {
+        this.grid = this.getView().getComponent('grid');
```

Comment:
Maybe we should just add two methods `getGrid` and `getChart`. I think it would look better than instance variables.